spec.md

```diff
@@ -0,0 +1,238 @@
+
+Authoring an OCI Artifacts involves the following steps:
+
+- Define a unique type
+- Define the format for other tools to operate upon the type
+- Define human elements, such as an icon and localized string to be displayed to users
+- Optionally, publish as a [well-known type][def-well-known-types] for registries to consume
+
+Defining a unique type, string and logo; artifacts can be represented as the following:
+
+|Icon|Artifact|`config.mediaType`|
+|-|-|-|
+|<img src="https://github.com/opencontainers/artwork/blob/master/oci/icon/color/oci-icon-color.png?raw=true" width=30x>|[OCI Image][image-spec]|`application/vnd.oci.image.config.v1+json`|
+|<img src="https://github.com/helm/helm-www/blob/master/themes/helm/static/img/apple-touch-icon.png?raw=true" width=30x>|[Helm Chart](https://helm.sh)|`application/vnd.cncf.helm.chart.config.v1+json`|
+|<img src="https://github.com/sylabs/singularity/blob/master/docs/logos/singularity_v3.png?raw=true" width=30x>|[Singularity][singularity], by [Sylabs][sylabs]|`application/vnd.sylabs.sif.config.v1+json`|
+
+## Defining a Unique Artifact Type
+
+Defining the unique type involves uniqueness for computer processing, and uniqueness for humans.
+
+For computer processing, artifacts are defined by setting `manifest.config.mediaType` to a globally unique value.
+
+> **Note:** The `config.mediaType` of `application/vnd.oci.image.config.v1+json` is reserved for artifacts intended to be instanced by [docker][docker] and [containerd][containerd].  
+*Each artifact MUST have its own unique type.*
+
+ The following `mediaType` format is used to differentiate the type of artifact:
+
+`application/vnd.`[org|company]`.`[objectType]`.`[optionalSubType]`.config.`[version]`+json`
+
+- **`org|company`** - represents an open source foundation (`oci`, `cncf`) or a company (`microsoft`, `ibm`).
+- **`objectType`** - a value representing the short name of the type. The combination of the org|company and object type should provide human identification and uniqueness for computers indexing.
+- **`optionalSubType`** - provides additional extensibility of an `objectType`
+- **`version`** - provides artifact authors the ability to revision their schemas and formatting, enabling tools to identify which format they will process.
+
+## Defining Supported Layer Types
```

Comment:
What is this section getting at?  Is it just "media-type authors, remember you can define Merkle trees and not just single files"?  If so, maybe an informative hint separate from the normative spec?

Reply:
Think of this as defining the file extension. It’s a format that provides more flexibility then just .someting.